src/probnum/diffeq/odefiltsmooth/odefiltsmooth.py

```diff
@@ -21,22 +21,29 @@
 import numpy as np
 
 import probnum.filtsmooth as pnfs
+import probnum.random_variables as pnrv
 from probnum.diffeq import steprule
+from probnum.diffeq.ode import IVP
 from probnum.diffeq.odefiltsmooth import ivp2filter
 from probnum.diffeq.odefiltsmooth.ivpfiltsmooth import GaussianIVPFilter
 
 
 def probsolve_ivp(
-    ivp,
-    method="eks0",
-    which_prior="ibm1",
-    atol=None,
-    rtol=None,
+    f,
+    t0,
+    tmax,
+    y0,
+    df=None,
+    method="EK0",
+    dense_output=True,
+    which_prior="IBM2",
+    atol=1e-2,
+    rtol=1e-2,
     step=None,
-    firststep=None,
-    **kwargs
+    driftspeed=None,
+    lengthscale=None,
```

Comment:
no more kwargs :) (of you're wondering why None as a default instead of, like, 1.0, see below)